charts/stable/traefik/templates/middlewares/theme-park.yaml

```diff
@@ -0,0 +1,23 @@
+{{- $values := .Values }}
+{{- $namespace := ( printf "ix-%s" .Release.Name ) }}
+{{- if or ( not .Values.ingressClass.enabled ) ( and ( .Values.ingressClass.enabled ) ( .Values.ingressClass.isDefaultClass ) ) }}
+{{- $namespace = "default" }}
+{{- end }}
+{{ range $index, $middlewareData := .Values.middlewares.redirectScheme }}
```

Comment:
```suggestion
{{- range $index, $middlewareData := .Values.middlewares.redirectScheme }}
```

Reply:
Should this need to be added on the rest of middlewares aswell?